doc-examples/hibernate-reactive-example-java/build.gradle

```diff
@@ -22,8 +22,14 @@ dependencies {
     implementation "io.micronaut.reactor:micronaut-reactor"
     compileOnly mnSerde.micronaut.serde.api
     runtimeOnly "ch.qos.logback:logback-classic"
-    runtimeOnly libs.drivers.vertx.pg
+    runtimeOnly libs.drivers.vertx.mysql
```

Comment:
Why the switch from Postgres?

Reply:
Native tests fail when `vertx-pg-client` is used because of the following GraalVM bug: `https://github.com/oracle/graal/issues/5510`. Because of the same reason the `hibernate6-reactive-postgres` native test has been disabled in the `micronaut-sql` too.